internal/blockchain/tezos/tezos.go

```diff
@@ -283,6 +283,10 @@ func (t *Tezos) RemoveFireflySubscription(ctx context.Context, subID string) {
 }
 
 func (t *Tezos) ResolveSigningKey(ctx context.Context, key string, intent blockchain.ResolveKeyIntent) (resolved string, err error) {
+	// Key is always required
+	if key == "" {
```

Comment:
Could use confirmation from @denisandreenko on if this is correct.

Reply:
This is preserving existing behavior (the empty string check has been pushed from the identity manager down to the blockchain plugins) - but I'm happy to tweak if there are protocol-specific rules (such as how Ethereum allows you to query without a key).

Reply:
Yes, this is correct.

Actually it gives error later at formatTezosAddress if key is empty string, but MsgNodeMissingBlockchainKey looks more specific which is better I believe. 

Thanks for catching this.

Reply:
Looks good to me